haproxy/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # CHANGELOG - haproxy
 
+1.1.0 / 2017-06-08
```

Comment:
Fixed - marked as "Unreleased", removed the corresponding version bump from `manifest.json`.